app/components/GettingStartedModal/installation/process.ts

```diff
@@ -60,8 +60,8 @@ export const updateSecret = async (manager: Manager, secret: SecretInstallationT
 
 // TODO(RD-2086): use common functions to upload blueprints
 export const uploadBlueprint = async (manager: Manager, blueprint: BlueprintInstallationTask) => {
-    const waitingTimeout = 5 * 60; // ~5 min = (N x 5s sleeps) + requests
-    const stepSleep = 5; // 5s
+    const waitingTimeout = 5 * 60; // ~5 minutes
+    const stepSleep = 5; // 5 seconds
```

Comment:
How about going further and adding the unit to the variable name?
```suggestion
    const waitingTimeoutSecs = 5 * 60;
    const stepSleepSecs = 5;
```

Reply:
updated